components/cluster/command/start.go

```diff
@@ -63,31 +47,8 @@ func newStartCmd() *cobra.Command {
 }
 
 func startCluster(clusterName string, options operator.Options) error {
-	logger.EnableAuditLog()
-	log.Infof("Starting cluster %s...", clusterName)
-	metadata, err := spec.ClusterMetadata(clusterName)
-	if err != nil && !errors.Is(perrs.Cause(err), meta.ErrValidate) {
-		return err
-	}
-
-	t := task.NewBuilder().
-		SSHKeySet(
-			spec.ClusterPath(clusterName, "ssh", "id_rsa"),
-			spec.ClusterPath(clusterName, "ssh", "id_rsa.pub")).
-		ClusterSSH(metadata.Topology, metadata.User, gOpt.SSHTimeout).
-		ClusterOperate(metadata.Topology, operator.StartOperation, options).
-		UpdateTopology(clusterName, metadata, nil).
-		Build()
-
-	if err := t.Execute(task.NewContext()); err != nil {
-		if errorx.Cast(err) != nil {
-			// FIXME: Map possible task errors and give suggestions.
-			return err
-		}
-		return perrs.Trace(err)
-	}
-
-	log.Infof("Started cluster `%s` successfully", clusterName)
-
-	return nil
+	return deployer.StartCluster(clusterName, options, func(b *task.Builder, metadata spec.Metadata) {
```

Comment:
Most of the logics have been removed from `startCluster`, I think we can inline this statement into `RunE` directly.